src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -115,6 +115,9 @@ public class DataServiceContext
 #if !PORTABLELIB // Timeout not available
         /// <summary>time-out value in seconds, 0 for default</summary>
         private int timeout;

```

Comment:
what's the difference between "timeout" and "readWriteTimeout" ?

Reply:
timeout  is the number of milliseconds to wait before the request(basically the connection) timesout while readWriteTimeout is the number of milliseconds to wait before the writing/reading of a request/response timeouts.